images/virtualization-artifact/werf.inc.yaml

```diff
@@ -32,5 +32,5 @@ shell:
   - export GOOS=linux
   - export CGO_ENABLED=0
   - export GOARCH=amd64
-  - go build -v -a -o virtualization-controller ./cmd/virtualization-controller
+  - go build -tags EE -v -a -o virtualization-controller ./cmd/virtualization-controller
```

Comment:
```suggestion
  - go build -tags {{ .MODULE_EDITION }} -v -a -o virtualization-controller ./cmd/virtualization-controller
```

Reply:
Since PR [#529](https://github.com/deckhouse/virtualization/pull/529) has been merged, please use the `MODULE_EDITION` variable in the build to build different editions